build.gradle

```diff
@@ -20,7 +20,11 @@ dependencies {
     testCompileOnly 'org.projectlombok:lombok:1.18.16'
     testAnnotationProcessor 'org.projectlombok:lombok:1.18.16'
     runtimeOnly 'mysql:mysql-connector-java'
+<<<<<<< HEAD
```

Comment:
Looks likes you forgot to resolve this conflict and remove the git version tags here.